artemis-protocols/artemis-amqp-protocol/src/test/java/org/apache/activemq/artemis/protocol/amqp/broker/AMQPMessageTest.java

```diff
@@ -156,6 +157,72 @@ public void testCreateMessageForPersistenceDataReload() throws ActiveMQException
       assertEquals(TEST_TO_ADDRESS, message.getAddress());
    }
 
+   @Test
```

Comment:
The tests seem a bit simplistic for such a change, what about annotations sections with many different annotations or annotations that don't match what you are looking for or are very very close but not quite what you are looking for etc.

Reply:
In this case it would lead to a false positive, that's ok in our case, given that is very unlikely that both values or keys are named similarly to a specific existing symbol.

Reply:
Probably I have understood what you mean and yes agree, I can improve the coverage with other cases :)